### PR TITLE
Bug 1210466 - Fix failure in test_call_log_groups.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6c003c52477f0d3e5fc4da9b622bc32d2476a9c6

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_call_log_groups.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class TestCallLogGroups(GaiaTestCase):
     def setUp(self):
 
         GaiaTestCase.setUp(self)
+        self.data_layer.set_setting('time.timezone', 'Europe/Istanbul')
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_call_log_groups.py]: I'd use UTC, to make sure of the timestamp given.

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_call_log_groups.py]: You mean, make use of the timezone similar as in London, UK? Then I would get the error as mentioned in https://bugzilla.mozilla.org/show_bug.cgi?id=1210466#c4 .

Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_call_log_groups.py]: Really? Why does Istanbul work then? It's UTC+3

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_call_log_groups.py]: I don't know, I wish I knew.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=31bbcb95e31e72438f240328d6fe2d296c94adab

**Pull request merged**